tests/telemetry/telemetry_utils.py

```diff
@@ -77,6 +77,19 @@ def restore_telemetry_forpyclient(duthost, default_client_auth):
         duthost.service(name="telemetry", state="restarted")
 
 
+def check_gnmi_cli_running(ptfhost):
+    program_list = ptfhost.shell("ps aux | grep py_gnmicli.py")["stdout"]
+    matches = re.findall('python /root/gnxi/gnmi_cli_py/py_gnmicli.py', program_list)
+    return len(matches) > 0
+
+
+def fetch_json_ptf_output(output, match_no):
+    match = re.findall('json_ietf_val: \"(.*)\"', output)
```

Comment:
>re [](http://example.com/codeflow?start=12&length=2)

Instread of regex search, can we use json parser?

Reply:
The ptf output does not come cleanly such that a json parser will work. I use regex search to easily search for the part of the output I want.